src/rest.js

```diff
@@ -65,11 +65,16 @@ function del(config, auth, className, objectId) {
       return find(config, Auth.master(config), className, {objectId: objectId})
       .then((response) => {
         if (response && response.results && response.results.length) {
-          response.results[0].className = className;
-
+          const firstResult = response.results[0];
+          firstResult.className = className;
+          if (className == '_Session' && !auth.isMaster) {
```

Comment:
prefer === i should prolly get that in lint....surprised it isn't already there.